src/index.html

```diff
@@ -5,8 +5,393 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="header__logo"
+      >  
+        <img
+          class="logo_img"
+          src="images/logo.png"
+          alt="MOYO"
+        >
+      </a>  
+      <nav class="nav">
```

Comment:
add an empty line between _multiline_ siblings. Check all over the document
```suggestion
      </a>  
      
      <nav class="nav">
```